### PR TITLE
Add empty chatroom message when user first logs into application 

### PR DESCRIPTION
**Pull request by edwjchen**
## Checklist

- [ ] Did you write tests for your code?
- [x] Did all the tests pass locally?

## Trello Ticket

n/a

## Short Description

Same as header!

## Reviewer Notes

Anything special you want the reviewer to know?

### CODE REVIEW

**Review by albinencio — APPROVED** [review]
LGTM

File: src/modules/ChatRoom.js [hunk 1]
```diff
@@ -346,23 +346,29 @@ class ChatRoom extends React.Component {
...
+                  ))
+                )}
+                <span ref={this.dummy}></span>
+              </div>
+              <ChatInput
+                message={this.state.message}
+                handleChange={this.handleChangeInput}
+                handleSubmit={this.handleSubmit}
+              />
             </div>
```
Comment (Joyfal) [inline src/modules/ChatRoom.js:371]: I would prefer to not change the position of <div>, the new structure does not make much sense (now that "chat-messages" is a child div inside "chat-person")

**Review by Joyfal — CHANGES_REQUESTED** [review]
One NIT

File: src/modules/ChatRoom.js [hunk 2]
```diff
@@ -346,23 +346,29 @@ class ChatRoom extends React.Component {
...
+                    <ChatMessage
+                      key={i}
+                      message={msg}
+                      username={this.state.user.username}
+                    />
+                  ))
+                )}
+                <span ref={this.dummy}></span>
+              </div>
+              <ChatInput
```
Comment (Joyfal) [inline src/modules/ChatRoom.js:366]: why don't we also disable the chatInput if we are in an empty room?

Comment (edwjchen) [reply src/modules/ChatRoom.js:367]: Oo good idea

File: src/modules/ChatRoom.js [hunk 3]
```diff
@@ -346,17 +346,23 @@ class ChatRoom extends React.Component {
...
+                  this.state.messages.map((msg, i) => (
+                    <ChatMessage
+                      key={i}
+                      message={msg}
+                      username={this.state.user.username}
+                    />
+                  ))
+                )}
+                <span ref={this.dummy}></span>
+              </div>
```
Comment (Joyfal) [inline src/modules/ChatRoom.js:365]: I think the div is still in the wrong place. Move this div at 365 to after 348 should fix it.

**Comment by edwjchen** [comment]
Disabled and enabled view

image
image

**Review by Joyfal — APPROVED** [review]
Great stuff!

File: src/modules/ChatRoom.js [hunk 4]
```diff
@@ -30,6 +30,7 @@ class ChatRoom extends React.Component {
       users: [],
       keyword: "",
       roomName: "Chat Room",
+      roomIsEmpty: true,
```
Comment (macedo22) [inline src/modules/ChatRoom.js:33]: Do we need this on top of `roomId`? Unless I'm overlooking something, `roomId == null` indicates a nonexistent room, so couldn't we just use that to determine if we are in an empty room? When it gets sent in `enterRoom`, I think that indicates the same thing as setting `roomIsEmpty == false`. Let me know your thoughts! I could be missing something

**Review by macedo22 — COMMENTED** [review]
I think we might not need `roomIsEmpty` but let me know

File: src/modules/ChatRoom.js (hunk 4 above)
Comment (Joyfal) [reply src/modules/ChatRoom.js:33]: Good point, I will change it.

**Review by macedo22 — APPROVED** [review]

**Pull request merged**